### PR TITLE
Potential Code Refactor

### PR DESCRIPTION
**Pull request by tusharnankani**

### CODE REVIEW

**Comment by tusharnankani** [comment]
The code could be refactored more, but I did not know if they would be required later. There were a few unused imported modules and variables.

**Comment by ankushbhardwxj** [comment]
@tusharnankani - Thanks for making the PR. Can you please resolve the conflicts ?

**Comment by tusharnankani** [comment]
Fixed 👍🏼

File: .deepsource.toml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version = 1
+
+[[analyzers]]
+name = "python"
+enabled = true
+
+  [analyzers.meta]
+  runtime_version = "3.x.x"
```
Comment (ankingcodes) [inline .deepsource.toml]: @tusharnankani - Did you setup deep source for this project ?

**Review by ankingcodes — CHANGES_REQUESTED** [review]

**Comment by ankushbhardwxj** [comment]
@tusharnankani - on reviewing your PR, I found that that only two lines of code were altered, for which it is difficult for me to give you points. There are many more areas in the codebase which require refactoring. You may go ahead and do some more. For example, using the function declared in `CodemonParse.py` inside of  `codemon.py` since they are related and many more. 

Let me know if you're interested to do some work, else we may either merge this one but you won't get much points for NWoC ?

**Comment by tusharnankani** [comment]
> The code could be refactored more, but I did not know if they would be required later. There were a few unused imported modules and variables.

I mentioned this earlier^ So let me know, I could optimize the code more.

**Comment by tusharnankani** [comment]
> > The code could be refactored more, but I did not know if they would be required later. There were a few unused imported modules and variables.
> 
> I mentioned this earlier^ So let me know, I could optimize the code more.


HI @ankingcodes 
Any updates?

**Comment by tusharnankani** [comment]
> > > The code could be refactored more, but I did not know if they would be required later. There were a few unused imported modules and variables.
> > 
> > 
> > I mentioned this earlier^ So let me know, I could optimize the code more.
> 
> HI @ankingcodes
> Any updates?

@ankingcodes you can merge it under nwoc and kwoc 👍🏻

**Pull request closed without merging**